packages/react-components/src/index.ts

```diff
@@ -15,6 +15,7 @@ export { StatusTimeline } from './components/status-timeline';
 export { Status } from './components/status/status';
 export { Tooltip } from './components/tooltip';
 export { KnowledgeGraph } from './components/knowledge-graph';
+export { Gauge } from './components/gauge/gauge';
```

Comment:
do not export until the gauge is fully complete and tested
ideally in several pull requests and not one large PR 🙏